tests/spicy/output/vector/reserve-known-elements.spicy

```diff
@@ -1,7 +1,7 @@
 # @TEST-DOC: Checks that a reserve call gets generated for vectors of known size
 #
 # @TEST-REQUIRES: which FileCheck
-# @TEST-EXEC: spicyc -p %INPUT | FileCheck %INPUT
+# @TEST-EXEC: spicyc -g -p %INPUT | FileCheck %INPUT
```

Comment:
`-g` disables basically every cleanup, let's maybe avoid doing that. We should be fine if we match on `# CHECK: {{[a-z_]+}}.reserve(128 <= 100 ? 128 : 100)` instead which seems localized enough.